SPECS/LICENSES-AND-NOTICES/LICENSE-LGPLv2.md

```diff
@@ -0,0 +1,503 @@
+### GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Why do we need this file here?

Reply:
The spec file for ceph was taken from Ceph source, under the LGPLv2 license.
`Initial CBL-Mariner import from Ceph source (license: LGPLv2.1)`
I have added the license in the license.json file:
`{`
           ` "origin": "Ceph",`
         `   "license": "[LGPL2.1](https://github.com/ceph/ceph/blob/master/COPYING-LGPL2.1)",`
           ` "specs": [`
              `  "ceph"`
        `    ]`
      `  }`